go/libraries/doltcore/sqle/dtables/merge_status_table.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dtables
+
+import (
+	"context"
+	"io"
+	"strings"
+
+	"github.com/dolthub/go-mysql-server/sql"
+
+	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
+	"github.com/dolthub/dolt/go/libraries/doltcore/sqle/dsess"
+	"github.com/dolthub/dolt/go/libraries/doltcore/sqle/index"
+	"github.com/dolthub/dolt/go/libraries/utils/set"
+)
+
+// MergeStatusTable is a sql.Table implementation that implements a system table
+// which shows information about an active merge.
+type MergeStatusTable struct {
+	dbName string
+}
+
+func (s MergeStatusTable) Name() string {
+	return doltdb.MergeStatusTableName
+}
+
+func (s MergeStatusTable) String() string {
+	return doltdb.MergeStatusTableName
+}
+
+func (s MergeStatusTable) Schema() sql.Schema {
+	return []*sql.Column{
+		{Name: "is_merging", Type: sql.Boolean, Source: doltdb.MergeStatusTableName, PrimaryKey: false, Nullable: false},
+		{Name: "source", Type: sql.Text, Source: doltdb.MergeStatusTableName, PrimaryKey: false, Nullable: true},
+		{Name: "source_commit", Type: sql.Text, Source: doltdb.MergeStatusTableName, PrimaryKey: false, Nullable: true},
+		{Name: "target", Type: sql.Text, Source: doltdb.MergeStatusTableName, PrimaryKey: false, Nullable: true},
```

Comment:
Isn't this just `active_branch()`?

Reply:
Correct, target is just the active branch. Do you think we should name it differently than `target`?